tests/replicaset_test.go

```diff
@@ -407,7 +406,7 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 
 	It("should replace a VMI immediately when a virt-launcher pod gets deleted", func() {
 		By("Creating new replica set")
-		template := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
+		template := libvmi.NewCirros()
```

Comment:
Replacing `NewRandomVMIWithEphemeralDiskAndUserdata` with `NewCirros` is not direct imho.

Reply:
Ok. Do you know what is the difference? What parameters should I add?

Reply:
Network binding is the difference. While it doesn't affect the test I would prefer to at least do this in 2 commits(One directly translates this to libvmi and in the second commit remove the binding)